src/backends/torch/torchsolver.cc

```diff
@@ -277,6 +277,25 @@ namespace dd
     // reload solver if asked for and set it value accordingly
     if (ad_mllib.has("resume") && ad_mllib.get("resume").get<bool>())
       {
+        if (mlmodel._sstate.empty())
+          {
+            std::string msg
+                = "resuming a model requires a solverstate file in model "
+                  "repository";
+            _logger->error(msg);
+            throw MLLibBadParamException(msg);
+          }
+        else
+          try
+            {
+              return load(mlmodel._sstate, main_device);
+            }
+          catch (std::exception &e)
+            {
+              this->_logger->error("Failed to load solver state");
```

Comment:
maybe log the name of the directory in which the file should be found ?

Reply:
ok